src/main/resources/assets/chisel/lang/en_US.lang

```diff
@@ -1878,3 +1878,15 @@ tile.glotek.12.desc=Cranberry
 tile.glotek.13.desc=Brink Pink
 tile.glotek.14.desc=Dark Salmon
 tile.glotek.15.desc=Harvest Gold
+
+#Froglight
+tile.chisel.froglight.name=Froglight
+tile.froglight.0.desc=Pearlescent
+tile.froglight.1.desc=Ocher
+tile.froglight.2.desc=Verdant
+tile.froglight.3.desc=Osmium
+tile.froglight.4.desc=Wolframsteel
+tile.froglight.5.desc=Ferric
+tile.froglight.6.desc=Titanic
+tile.froglight.7.desc=Alumium
```

Comment:
Typo: "Alumium"

Reply:
That was intentional actually, I already felt like i cheaped out on osmium by not giving it a fancy name, and "[Alumium](https://en.wiktionary.org/wiki/alumium)" apparently used to be a way to refer to Alumin(i)um